packages/db/package.json

```diff
@@ -40,7 +40,8 @@
     "lodash": "^4.17.20",
     "pg": "^8.3.0",
     "snake-camel": "^1.0.6",
-    "uuid": "8.3.1"
+    "uuid": "8.3.1",
+    "crypto-random-string": "^3.2.0"
```

Comment:
this is a nit but can we alphabetize this?

Reply:
b572abd2a7995d7b31228c566415288225d62778